src/cmd/build.js

```diff
@@ -103,15 +111,49 @@ async function defaultPackageCreator({
   let packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   let extensionPath = path.join(artifactsDir, packageName);
-  let stream = createWriteStream(extensionPath);
 
-  stream.write(buffer, () => stream.end());
+  let destinationFileExists = await fs.stat(extensionPath)
+    .then((stat)=>{
+      return stat.isFile();
+    })
+    .catch(()=>{
```

Comment:
Whenever you catch and ignore an error you will also need to add a `log.debug()` statement showing the original error. This will help us debug a situation where the wrong assumption was made about the caught error.